index.bs

```diff
@@ -111,17 +111,18 @@ permission-related algorithms and types are defined as follows:
   run these steps:
 
   1. Let |entry| be |desc|.{{FileSystemPermissionDescriptor/handle}}'s [=FileSystemHandle/entry=].
-  1. If |entry| represents a [=/file system entry=] in an [=origin private file system=],
-    this descriptor's [=permission state=] must always be {{PermissionState/"granted"}}.
+  1. If |entry| represents a [=/file system entry=] in a [=/bucket file system=],
+     this descriptor's [=permission state=] must always be
+     "{{PermissionState/granted}}".
   1. Otherwise, if |entry|'s [=file system entry/parent=] is not null, this descriptor's [=permission state=] must be
-    equal to the [=permission state=] for a descriptor with the same {{FileSystemPermissionDescriptor/mode}},
-    and a {{FileSystemPermissionDescriptor/handle}} representing |entry|'s [=file system entry/parent=].
-  1. Otherwise, if |desc|.{{FileSystemPermissionDescriptor/mode}} is {{"readwrite"}}:
+     equal to the [=permission state=] for a descriptor with the same {{FileSystemPermissionDescriptor/mode}},
+     and a {{FileSystemPermissionDescriptor/handle}} representing |entry|'s [=file system entry/parent=].
+  1. Otherwise, if |desc|.{{FileSystemPermissionDescriptor/mode}} is "`readwrite`":
```

Comment:
"{{FileSystemPermissionMode/readwrite}}"

Reply:
Sorry, I meant to add a comment here asking whether to use the string in the enum value in this case. Thanks for clearing that up!